docs/src/pages/components/transitions/transitions.md

```diff
@@ -81,6 +58,33 @@ This example also demonstrates how to delay the enter transition.
 
 {{"demo": "pages/components/transitions/SimpleZoom.js", "bg": true}}
 
+## Child requirement
+
+- **Forward the style** : To better support server rendering, Material-UI provides a `style` prop to the children of some transition components, (Fade, Grow, Zoom, Slide). The `style` prop must be applied to the DOM for the animation to work as expected.
+- **Forward the ref** : The transition components require the first child element to forward its ref to the DOM node. For more details about ref, check out [Caveat with refs](/guides/composition/#caveat-with-refs)
+- **Single element** : The transition components require only 1 child element (`React.Fragment` is not allowed).
+
+```jsx
+// The `props` object contains a `style` prop.
+// You need to provide it to the `div` element as shown here.
+const MyComponent = React.forwardRef(({ style, ...props }, ref) {
+  return (
+    <div ref={ref} {...props} style={style}>
```

Comment:
We already have a comment about forwarfing the `style` prop. I don’t think we need to emphasize again, especially because the `style` prop is not handled in the component.